src/sidebar/helpers/version-data.js

```diff
@@ -53,13 +53,13 @@ export default class VersionData {
    * @return {string} - Single, multiline string representing current version data
    */
   asFormattedString() {
-    let versionString = '';
-    for (let prop in this) {
-      if (Object.prototype.hasOwnProperty.call(this, prop)) {
-        versionString += `${prop}: ${this[prop]}\r\n`;
-      }
-    }
-    return versionString;
+    return `Version: ${this.version}
```

Comment:
Thanks for refactoring this. I think I'm guilty of trying to make it overly-generic in its first iteration.